docs/semgrep-supply-chain/overview.md

```diff
@@ -54,15 +54,12 @@ your critical/high-severity findings as either reachable or unreachable.
   * If there's a code pattern in the codebase that matches the vulnerability
     definition, the finding is flagged as **reachable**.
       * A finding is **always reachable** if the only way to fix the vulnerability is to upgrade the dependency. Semgrep strongly recommends upgrading the dependencies involved for these findings.
-      * A finding is **conditionally reachable** if Semgrep finds a way to reach it when scanning your code when certain conditions are met.
-  * If you don't use the vulnerable piece of code of the library or package
-  imported, the finding is flagged as **unreachable**.
+      * A finding is **conditionally reachable** if the vulnerability can be exploited when specific conditions are met. Semgrep Supply Chain can determine if the vulnerability is reachable, but such findings require manual review to determine whether those conditions have been met.
```

Comment:
> Semgrep Supply Chain can determine if the vulnerability is reachable, but such findings require manual review to determine whether those conditions have been met.

This sounds a little self-contradicting? If Semgrep "can determined if the vulnerability is reachable", then doesn't that mean it doesn't need manual review? I understand that this statement/type of finding was for cases beyond code reachability, Misha's example was I think some kind of situation where the function had to be called and the operating system had to be some specific thing. (I could be recalling things wrong, so I'm spelling it out for corrective purposes.)

Maybe something like:

> The finding is reachable if in addition to the dataflow reachability in code, another factor, such as the environment's operating system, must be met. Semgrep cannot determine such factors, so conditionally reachable findings require manual review.